cypress.config.js

```diff
@@ -0,0 +1,16 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: "https://social-media-client-msiksnis.vercel.app",
```

Comment:
```suggestion
    baseUrl: "/",
```

You should be testing code local to your branch, not remote code that has already been merged!